src/lib/components/nav/PageHeader.svelte

```diff
@@ -0,0 +1,19 @@
+<script lang="ts">
+  import { page } from "$app/stores";
+  import SetPageTitle from "$lib/components/nav/SetPageTitle.svelte";
+  import { twMerge } from "tailwind-merge";
+
+  export let title: string | undefined;
+  let clazz = "";
+  export { clazz as class };
+</script>
+
+<SetPageTitle {title} />
+
+<!-- If it's an app, the page title is shown at the top, in the header -->
+{#if !$page.data.isApp}
```

Comment:
The comment says `"If it's an app"`, but the condition says `if not page.data.isApp`.

Reply:
Well, the comment doesn't describe the code below, but instead a comment to why this logic is there. I think the logic speaks for itself. "If not an app, show large h1 text with title". But why? That is described in the comment

Reply:
Aha of course. I'll clarify it slightly.
```suggestion
<!-- If it's an app, the page title is shown at the top instead, in the header -->
{#if !$page.data.isApp}
```